omod/src/main/webapp/mdrtbListPatients.jsp

```diff
@@ -188,10 +188,10 @@
 													</a>
 												</openmrs:extensionPoint>
 											</td>
-											<td class="patientTable" style="white-space:nowrap;">${p.patientIdentifier.identifier}</td>
-											<td class="patientTable" >${p.familyName}, ${p.givenName} ${p.middleName}</td>
-											<td class="patientTable" style="white-space:nowrap;">${p.age}</td>
-											<td class="patientTable" style="white-space:nowrap;">${p.gender}</td>
+											<td class="patientTable" style="white-space:nowrap;"><c:out value="${p.patientIdentifier.identifier}"/></c:out></td>
```

Comment:
can you remove the last </c:out> on this line

Reply:
Ooohhh am working on it

Reply:
i have fixed the issue  Thanks @HerbertYiga